docs/cluster-providers/generic-cluster-provider.md

```diff
@@ -29,16 +33,29 @@ const clusterProvider = new blueprints.GenericClusterProvider({
             diskSize: 50
         },
         {
-            id: "mng2-custom",
-            instanceTypes: [new InstanceType('m5.2xlarge')],
+            id: "mng2-launchtemplate",
+            instanceTypes: [new ec2.InstanceType('t3.large')],
             nodeGroupCapacityType: CapacityType.SPOT,
-            customAmi: {
+            desiredSize: 2,
+            minSize: 2,
+            maxSize: 3, 
+            launchTemplate: {
```

Comment:
i assume i can use tags with the standard EKS optimized AMI as opposed to custom AMI, correct? That is the use case we want to promote, so we can just add tags to the first MNG in the cluster.